client/rest.go

```diff
@@ -1460,15 +1445,29 @@ func (restClient *RestClient) getFeeRate() gin.HandlerFunc {
 	}
 }
 
-func avg(arr []store.MempoolRecord) int {
-	total := 0
-	for _, value := range arr {
-		total += value.Category
-	}
-	if total == 0 {
-		return 0
+func fetchMempool(mempool sync.Map) int64 {
```

Comment:
looks like we can cover this func with unittest

Reply:
ok